### PR TITLE
Nathan/new announcements page

### PR DESCRIPTION
**Pull request by nathanzzhang**
- Created static Announcements page with text "Check back here for announcements!"
- Added Announcements tab to menu bar

#165

### CODE REVIEW

File: src/components/AnnouncementsPage/Announcements.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+export default function AnnouncementsPage() {
+	const theme = useTheme();
+
+	return (
+		<>
+			<Typography variant='h4' component='h1'
+				style={{ fontWeight: theme.typography.fontWeightBold,
+					paddingTop: theme.spacing(37), // initial value 4
+					paddingBottom: theme.spacing(37), // initial value 4
```
Comment (jodymlin) [inline src/components/AnnouncementsPage/Announcements.js:14]: I can see that you used this to vertically center the text on the screen and also make text fill up the entire screen (so the footer doesn't look awkward in the middle of the screen). Screen sizes differ, however, and this results in the page sometimes taking up too much height and sometimes still taking up too little height. 

I just made new fix that makes all pages take up a minimum height of the size of the screen (regardless of how much content is on the page) so you don't need to have this large padding anymore. Feel free to just use the original `theme.spacing(4)` padding. It's not a big deal if the text is not vertically centered.

**Review by jodymlin — CHANGES_REQUESTED** [review]
Some minor changes requested but otherwise great job!

The fix I mentioned about the screen height is in pr #174 so don't worry about having to manually fill the page space.

File: src/components/AnnouncementsPage/Announcements.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import Typography from '@material-ui/core/Typography';
+import useTheme from '@material-ui/core/styles/useTheme';
+import Container from '@material-ui/core/Container';
+
+export default function AnnouncementsPage() {
+	const theme = useTheme();
+
+	return (
+		<Container>
```
Comment (jodymlin) [inline src/components/AnnouncementsPage/Announcements.js:11]:
```suggestion
		<Container maxWidth='md'>
```

All the other pages use `md` width containers, so let's make this `md` as well for consistency. This prop and its values can be found [here](https://mui.com/api/container/) on the MUI docs for `Container`

**Review by jodymlin — CHANGES_REQUESTED** [review]
One last minor change to make then looks good to me and I'll approve it 👍

File: src/components/AnnouncementsPage/Announcements.js (hunk 2 above)
Comment (nathanzzhang) [reply src/components/AnnouncementsPage/Announcements.js]: ah i see, thanks!

**Review by jodymlin — APPROVED** [review]

**Review by AnakinTrotter — APPROVED** [review]
LGTM!

**Pull request merged**